quack_api/quack_api.Test/UnitTests/RecommenderControllerUnitTests.cs

```diff
@@ -5,34 +5,43 @@
 using System.Linq;
 using System.Text;
 using System.Text.Json;
+using System.IO;
 using System.Threading.Tasks;
 using quack_api.Models;
 using quack_api.Controllers;
 using Microsoft.Extensions.Options;
 using quack_api.Enums;
+using Microsoft.Extensions.Configuration;
 
 namespace quack_api.Test.UnitTests
 {
     [TestClass]
     public class RecommenderControllerUnitTests
     {
-        
+        public static IConfiguration InitConfiguration()
```

Comment:
This method I would move to a utility class called GeneralUtil in a folder called Utilities